app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+
+    <string name="rom_settings">Configurações da ROM</string> <!-- Cadeia de caracteres de acessibilidade. Não se deve usar acrônimos (siglas) -->
+    <string name="label_rom_config_console">Sistema Inicializador</string>
+    <string name="label_rom_config_load_gba_rom">Carregar ROM de GBA</string>
+    <string name="label_rom_config_gba_rom_path">Caminho da ROM de GBA</string>
+    <string name="label_rom_config_gba_save_path">Salvar caminho de GBA</string>
+
+    <string name="theme_option_light">Claro</string>
+    <string name="theme_option_dark">Escuro</string>
+    <string name="theme_option_system_default">Padrão do Sistema</string>
+
+    <string name="notification_channel_background_tasks">Tarefas de segundo plano</string>
+
+    <string name="top_screen">Tela superior</string>
+    <string name="bottom_screen">Tela inferior</string>
+    <string name="input_a">A</string>
+    <string name="input_b">B</string>
+    <string name="input_x">X</string>
+    <string name="input_y">Y</string>
+    <string name="input_left">Esquerda</string>
+    <string name="input_right">Direita</string>
+    <string name="input_up">Cima</string>
+    <string name="input_down">Baixo</string>
+    <string name="input_l">L</string>
+    <string name="input_r">R</string>
+    <string name="input_start">Start</string>
+    <string name="input_select">Select</string>
+    <string name="input_lid">Abrir/Fechar Tampa</string>
+    <string name="input_pause">Pausa</string>
+    <string name="input_fast_forward">Avanço rápido (Alternar)</string>
+    <string name="input_toggle_soft_input">Alternar entrada de toque</string>
+    <string name="input_reset">@string/reset</string>
+    <string name="input_dpad">DPAD</string>
+    <string name="input_abxy_buttons">Botões ABXY</string>
+    <string name="input_swap_screens">Alternar Telas</string>
+    <string name="input_quick_save">Salvamento Rápido</string>
+    <string name="input_quick_load">Carregamento Rápido</string>
+
+    <string name="layouts">Layouts</string>
+    <string name="layouts_summary">Gerenciar configurações de personalizações de layout</string>
+    <string name="action_layouts_new">Novo layout</string>
+    <string name="default_layout_name">Padrão</string>
+    <string name="use_global_layout">Usar layout global</string>
+    <string name="layout_deleted">Layout deletado</string>
+
+    <string name="layout_editor">Editor de layout</string>
+    <string name="add_component">Adicionar componente</string>
+    <string name="no_more_components">Não há mais componentes para adicionar</string>
+    <string name="label_size">Tamanho:</string>
+    <string name="menu">Menu</string>
+    <string name="properties">Propriedades</string>
+    <string name="revert_changes">Reverter alterações</string>
+    <string name="reset_default">Redefinir para o padrão</string>
+    <string name="exit_without_saving">Sair sem salvar</string>
+    <string name="portrait_background">Fundo do Modo Retrato</string>
```

Comment:
This string has incorrect capitalization. Only the first letter should be in uppercase. The same applies to the 3 strings below this one.

Reply:
😅 I ended up doing it to get the end user's attention. I will correct.